webpack.config.js

```diff
@@ -0,0 +1,68 @@
+const webpack = require("webpack");
+const path = require("path");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const CleanWebpackPlugin = require("clean-webpack-plugin");
+
+const widgetConfig = {
+    entry: {
+        DeviceIdWidget: "./src/DeviceIdWidget/widget/DeviceIdWidget.ts",
+    },
+    output: {
+        path: path.resolve(__dirname, "dist/tmp"),
+        filename: "src/DeviceIdWidget/widget/DeviceIdWidget.js",
+        libraryTarget: "umd"
+    },
+    resolve: {
+        extensions: [ ".ts", ".js" ],
+        alias: {
+            "tests": path.resolve(__dirname, "./tests")
+        }
+    },
+    module: {
+        rules: [
+            { test: /\.ts$/, use: "ts-loader" }
+        ]
+    },
+    devtool: "source-map",
+        externals: [

```

Comment:
use regex to exclude all 
` externals: [ /^mxui\/|^mendix\/|^dojo\/|^dijit\// ],`

Dont exclude react... is it not used in the project for mx 6 widgets